testgrid/tgrun/pkg/runner/embed/runcmd.sh

```diff
@@ -439,7 +455,22 @@ function main() {
     if ! check_airgap ; then
         failureReason="airgap_violation"
     fi
-
+    i=0
+    while true
+    do
+        #cluster_ready
+        if [ "$(cluster_ready)" -eq 1 ]; then
+            echo "cluster is ready"
+            break
+        fi
+        echo "cluster is not ready"
+        i=$((i+1))
+        if [ $i -gt 5 ]; then
+            report_success "cluster_not_ready"
+            exit 0
+        fi
+        sleep 100
```

Comment:
you can leave this for now but i wonder if we will need to increase the timeout here and overall timeouts of the run